src/main/java/eu/greev/dcbot/ticketsystem/interactions/commands/Transfer.java

```diff
@@ -46,8 +46,23 @@ public void execute(Event evt) {
                     .queue();
             return;
         }
-
         Ticket ticket = ticketService.getTicketByChannelId(event.getChannel().getIdLong());
+        EmbedBuilder error = new EmbedBuilder()
+                .setColor(Color.RED)
+                .setFooter(config.getServerName(), config.getServerLogo());
+
+         if (ticket.getSupporter() == null) {
+            event.replyEmbeds(error.setDescription("You can not transfer a ticket which wasn't claimed!").build())
+                    .setEphemeral(true)
+                    .queue();
+            return;
+        } else if (ticket.getSupporter().equals(event.getUser())) {
```

Comment:
Since you `return` in the line before this, there is no need for an `else if` here, just use `if`.  
Furthermore, please add the ability for teammembers to overwrite this. It happens that a support for example goes on vacation or leaves the team before they finish all tickets. (If admins and above have this power, it should be okay)